acquia/acquia_cms
# ACMS-1424: Fix issues when enabling acquia_cms modules in isolation.

**Motivation**

Fixes [ACMS-1424](https://backlog.acquia.com/browse/ACMS-1424)

**Proposed changes**


**Alternatives considered**


**Testing steps**
Use this branch
`composer install`
`composer update`
Install site using `drush site:install minimal`
Install Acquia CMS Site Studio module using `drush in acquia_cms_site_studio -y`
Install Acquia CMS Search module using `drush in acquia_cms_search -y`
Now install any other module with site studio package ex ACMS Article using `drush in acquia_cms_article -y`
Verify that all packages imported related to Article content type
Verify All packages imported related to Search and Article views created using search index

**Merge requirements**
